custom/icds_reports/static/icds_reports/mobile/css/app.less

```diff
@@ -395,3 +395,6 @@
 .align-items-flex-end {
   align-items: flex-end;
 }
+.clickable-breadcrumb {
+  border-bottom: 2px solid white;
```

Comment:
Actually missed this, but would `text-decoration: underline` not be more appropriate?

Reply:
![Screenshot from 2020-03-30 10-25-30](https://user-images.githubusercontent.com/56248457/77877329-178acf80-7273-11ea-98ba-54fee19dc7e3.png)

It looked like this. I felt this is not emphasising. But you can take the call on what to use

Reply:
I think let's go with the "standard" version here for now unless anyone complains. Using `border-bottom` just feels a bit odd to me